common/common.mk

```diff
@@ -86,7 +88,7 @@ llvm-check: $(CLANG) $(LLC)
 
 $(OBJECT_LIBBPF):
```

Comment:
Do we need this rule in the Makefile - if we are moving to libxdp?

Reply:
I was originally hoping to use libbpf as a submodule of xdp-tools, but xdm-tools does not expose whether it is using the submodule or an installed libbpf. I followed what bpf-examples does which is to have its own libbpf as well as xdp-tools. There might be a better approach but it would require changes across at least two projects.